src/main/java/Entities/Song.java

```diff
@@ -1,4 +1,60 @@
 package Entities;
+import java.awt.image.BufferedImage;
+import java.util.List;
 
 public class Song {
+
+    private final int id;
+    private final String name;
+    private final List artistList;
+    private final int length;
+    private final String genre;
+    private final String fileLocation;
+    private final boolean isExplicit;
+    private final BufferedImage cover;
+
+
+    public Song(int id, String name, List artistList, int length, String genre,
+                String fileLocation, boolean isExplicit, BufferedImage cover){
+        this.id = id;
+        this.name = name;
+        this.artistList = artistList;
+        this.length = length;
+        this.genre = genre;
+        this.fileLocation = fileLocation;
+        this.isExplicit = isExplicit;
+        this.cover = cover;
+    }
+
+    public int getID(){
```

Comment:
Could you add a comment in the code (or add a brief response to this comment) explaining what the id is for? I assume database stuff, but I'm wondering whether I'd should/have to make use of the id variable.

Reply:
Yeah I'll do it ASAP, You'd likely have to use the ID by the way, its the unique identifier for a given song rather than using its actual name (see: different songs with the same name in practice)